Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Suspicious_user_access_to_multiple_resources_via_SSO.yml

```diff
@@ -0,0 +1,1472 @@
+id: silent-Suspicious user access to multiple resources via SSO
+version: -1
+name: silent-Suspicious user access to multiple resources via SSO
+description: |-
+  **This playbook addresses the following alert**:
+  - Suspicious user access to multiple resources via SSO.
+
+  **Playbook Stages**:
+
+  **Triage**:
+  - Collect additional alert data.
+
+  **Investigation**:
+  - **IP Reputation Check**:
+    - Look up the reputation of the IP address associated with the sign-in attempt.
+  - **Uncommon Target Application Ratio**:
+    - Calculate the percentage of accessed applications that are considered uncommon for the user.
+  - **Check for Azure Alerts**:
+    - Retrieve Azure security alerts linked to the user over the last 24 hours.
+  - **Check If User Is Risky**:
+    - Retrieve risk indicators from Azure and Cortex.
+    - Determine if the user has medium/high risk detections and capture relevant risk reasons.
+  - **Activity Duration Analysis**:
+    - Calculate the time window between the first and last activity in the alert.
+  - **Search for Related Alerts**:
+    - Look for other alerts related to the same user with suspicious access patterns (e.g., abnormal SSO activity).
+
+  - **Check for Suspicious Evidence**, including:
+    - IP address flagged as malicious.
+    - Confirmed Azure or Core risky detections.
+    - Relevant Azure security alerts.
+    - High ratio of uncommon applications accessed.
+    - Short sign-in activity window.
+    - Behavioral deviation in number of accessed apps compared to normal usage.
+    - Additional related alerts in the environment.
+
+  **Containment**:
+  - **Analyst Review**:
+    - Present findings and evidence to the analyst for review.
+    - If needed, the analyst can choose to disable the user.
+  - **Automatic Actions**:
+    - If the analyst disables the user, clear their active sessions as a containment measure.
+
+  **Requirements**:
+  Ensure the following integrations are configured:
+  - **Microsoft 365 Defender** for Azure user alerts.
+  - **Microsoft Graph Identity** for user disablement and session control.
+  - **Core** for alert context and user risk scores.
+  - **Threat Intelligence Integrations** for IP reputation analysis.
```

Comment:
is this a name of the integration?

Reply:
This is not a name of an integration since there are several integrations which support the ip reputation analysis (we use a generic command).